src/plugins/stake-plugins/thorchainSavers/tcSaversPlugin.ts

```diff
@@ -555,7 +557,7 @@ const stakeRequest = async (opts: EdgeGuiPluginOptions, request: ChangeQuoteRequ
     // 2. Only use UTXOs from the primary address (index 0)
     // 3. Force change to go to the primary address
     otherParams: { outputSort: 'targets', utxoSourceAddress, forceChangeAddress },
-    metadata: { name: 'Thorchain Savers', category: 'Transfer:Staking' }
+    metadata: { name: 'Thorchain Savers', category: `Transfer:Thorchain Savers ${sprintf(lstrings.transaction_details_stake_subcat_1s, currencyCode)}` }
```

Comment:
Is this string supposed to be in notes?

Reply:
No. This is a subcat string. Transfer categories should have subcategories that specify the unique "entity" or "account" that is receiving the funds. "Thorchain Savers Stake BTC" is a specific account that funds are being transferred to and from. Although admittedly the name would be better described as "Thorchain Savers Staked BTC"